application_integration/create_auth_config/main.tf

```diff
@@ -16,11 +16,21 @@
 
 provider "google" {}
 
-# [START application_integration_create_auth_config_auth_token]
```

Comment:
Moving this line means that you will no longer include the resource in the region tag. This may need to be changed later.